webapp/exercise_admin/static/exercise_admin/exercise-admin.js

```diff
@@ -45,8 +45,86 @@ $(document).ready(function() {
     $('.popup > div').click(function(event) {
         event.stopPropagation();
     });
+    $('input[type=text].exercise-tag-input').each(function () {
+        change_tag_width(this);
+        if (this.value === "") {
+            remove_tag(this);
+        }
+    });
 });
 
+function highlight_parent_li(input_elem) {
+    var input = $(input_elem);
+    var parent = input.parent();
+    parent.css({'background' : '#cec'});
+    input.focusout(function() {
+        if (input.val() === '') {
+            remove_tag(input[0]);
+        } else if (!parent.is(':hover')) {
+            parent.css({'background' : '#ddd'});
+        }
+    });
+    parent.hover(function() {
+        parent.css({'background' : '#cec'});
+    }, function() {
+        if (!input.is(':focus')) {
+            parent.css({'background' : '#ddd'});
+        }
+    });
+}
```

Comment:
Instead of tweaking the css styling from here, create separate classes in the .css-file and swap between them here. This keeps the styling and code separate and makes migration to SCSS easier.